### ISSUE TITLE
Recreate MediaRecorder after every stop()

### ISSUE THREAD

**Issue by floydback** [ai/audio-recorder-polyfill#28]
There is a difference in native and polyfill MediaRecorders behavior.

With native I can do:
```
recorder = new MediaRecorder(stream, options)
recorder.start()
recorder.stop()
recorder.start() // start recording second time
```

With polyfill this work with a bug, so I must do:
```
recorder = new MediaRecorder(stream, options)
recorder.start()
recorder.stop()
recorder = new MediaRecorder(stream, options) // need re-create recorder
recorder.start()
```

Bug: When I start recording with polyfill second time (without re-creating) the result audio has stuttering (and has incorrect duration ~x2). When I start recording third time this effect are increased.

Maybe it should be better to make polyfill behave as native MediaRecorder?

**Comment by ai** [comment]
Sure. Do you want to send PR?

**Comment by floydback** [comment]
I never do it but I try

**Comment by floydback** [comment]
Done. The **createScriptProcessor** must called once, not each time calling start()

**Comment by ai** [comment]
Fixed by https://github.com/ai/audio-recorder-polyfill/pull/29

**Comment by ai** [comment]
Do you have a twitter account to mention you in release tweet?

**Comment by ai** [comment]
Released in 0.1.6.
